bin/wkhtmltopdf

```diff
@@ -10,6 +10,21 @@
 require 'rbconfig'
 require 'securerandom'
 require 'fileutils'
+require 'logger'
+
+def log_duration(type = :info, log_name)
+  log = Logger.new(STDERR)
```

Comment:
should it be STDOUT if it's just info?

Reply:
I tried STDOUT and crashed Puma. STDERR worked. The output with STDERR is like this (I changed the final message from "PDF wkhtmltopdf - Logger.new(STDERR)" to "PDF: chromium execution")

![Screen Shot 2022-05-27 at 4 55 50 PM](https://user-images.githubusercontent.com/87718108/170788408-b5931df2-6b3b-42b0-ae81-1a8def016d41.png)
.